chapters/annotations.tex

```diff
@@ -27,9 +28,72 @@ \section{Notation for Annotation Definitions}\label{notation-for-annotation-defi
 If the annotation is declared as an \lstinline!/*evaluable*/ parameter! the corresponding modifier is further restricted to be evaluable.
 If the annotation is declared as a \lstinline!/*literal*/ constant! the corresponding modifier is further restricted to be a literal value.
 
+
+\section{Semantic Restrictions of Annotation Syntax}\label{semantic-restrictions-of-annotation-syntax}
+
+The syntactic form of annotations, \lstinline[language=grammar]!annotation-clause!, uses the very generic \lstinline[language=grammar]!class-modification!.
```

Comment:
```suggestion
The syntactic form of annotations, \lstinline[language=grammar]!annotation-clause!, uses the generic \lstinline[language=grammar]!class-modification!.
```
I don't see why "very" helps here.
If we wanted to give an intuitive understanding we could say that they are generally "value-modifiers".

Reply:
OK, I removed the _very_, although I personally find `class-modification` _very_ generic considering the form of almost all valid annotations.  To compensate a little bit, I'm adding a cross reference to make it easier to see the generality in the grammar with one's own eyes.